infrastructure/tasks/10-functions_setup.ts

```diff
@@ -122,12 +128,18 @@ export const run = async () => {
         // to retrieve the master key
         { name: "AzureWebJobsSecretStorageType", value: "disabled" },
         { name: "WEBSITE_HTTPLOGGING_RETENTION_DAYS", value: "3" },
+        { name: "SCM_USE_FUNCPACK_BUILD", value: "1" },
         {
           name: "MESSAGE_CONTAINER_NAME",
           value: config.message_blob_container
         }
       ],
       connectionStrings: [
+        {
+          connectionString: process.env.SENDGRID_KEY,
```

Comment:
could you add a not about this part that must be updated on the story for moving to a different provider